### ISSUE TITLE
is it possible to have an extra field where certain % of expences can be added unto item cost price to biuld selling price automatically

### ISSUE THREAD

**Issue by trendcnet** [jekkos/opensourcepos#516]

**Comment by daN4cat** [comment]
You mean having a markup / margin %. Sounds a good idea. What would be the rounding policy then?

**Comment by trendcnet** [comment]
yes, markup % built on the unit cost price of each item to determine what will be the actual cost price (travelling expences and procurement expences added), it will be flexible to allow entry in % extra expences totalled and then added to unit price of each item to now have the actual cost price so as to build profitable sellting/retail price.

**Comment by daN4cat** [comment]
Agreed, but what rounding do you foresee? That's a key point. Once you add up based on % you may have 34.56 or 78.91... Those are not real retail prices. And then you get the classical 89.99 request.

So it's better to think about the rounding policy before implementing this feature.

**Comment by trendcnet** [comment]
This is quite true but after adding up and we are getting values like 34.56 as illustrated, one can decide to roundup the selling price manualy in a flexible way that will offer real retail price.

**Comment by daN4cat** [comment]
Socalculated with manual command (button press) and then manually rounded.

**Comment by kosimoff** [comment]
Also bulk editing purchasing and selling price. Increase or decrease some fixed value or percentage.
For example
Item 1: 5$
Item 2: 6$
Item 3: 7$

Select all and bulk edit. Increase selling price by 5%. Then it automatically increases all selected items. Also decrease. Nitnonly selling but also purchasing price. Not just by percentage but also by fixed value. For example decrease purchase price by 1$.

This is very useful if you have lots of items and your supplier suddenly increased their price, or decreased. Or you import the products from another country and currency exchange rates changed significantly.

**Comment by daN4cat** [comment]
Ref #1963

**Comment by WebShells** [comment]
Can be done by simple jscript values calculations if am not wrong... Do you want it as simple as that ?
